core/src/commonMain/kotlin/io/github/aag/core/domain/models/Stringified.kt

```diff
@@ -0,0 +1,5 @@
+package io.github.aag.core.domain.models
+
+interface Stringified {
```

Comment:
What's the purpose of this class? Is this some KMM specific stuff?

Reply:
Actually this interface is a filter value. Courses could be filtered by `City`, `CourseYear`, `CourseLevel` (these are all enums which are present in back-end response as separate fields). In UI they are three separate dropdown lists.      
Probably, I should rename this `Stringified` to `FilterItem` for better readability

Reply:
Done, renamed.